### PR TITLE
remove @Nullable annotations and dependency on com.google.code.findbugs:jsr305

### PR DESCRIPTION
**Pull request by dbyron0**
Another (in addition to https://github.com/testcontainers/testcontainers-java/pull/913) way to address #912.

### CODE REVIEW

**Comment by kiview** [comment]
This is the only location where we use `@Nullable?`

**Comment by dbyron0** [comment]
Apparently not:
```
$ GIT_PAGER="" git grep Nullable
core/src/main/java/org/testcontainers/containers/Container.java:        Optional<String> oldValue = Optional.ofNullable(getEnvMap().get(key));
core/src/main/java/org/testcontainers/containers/GenericContainer.java:import org.jetbrains.annotations.Nullable;
core/src/main/java/org/testcontainers/containers/GenericContainer.java:    @Nullable
core/src/main/java/org/testcontainers/containers/PortForwardingContainer.java:        return Optional.ofNullable(container)
core/src/main/java/org/testcontainers/dockerclient/DockerClientConfigUtils.java:            .ofNullable(DockerClientFactory.instance().runInsideDocker(
core/src/main/java/org/testcontainers/dockerclient/DockerClientProviderStrategy.java:import org.jetbrains.annotations.Nullable;
core/src/main/java/org/testcontainers/dockerclient/DockerClientProviderStrategy.java:                        @Nullable String throwableMessage = e.getMessage();
core/src/main/java/org/testcontainers/dockerclient/DockerClientProviderStrategy.java:                        @Nullable String rootCauseMessage = rootCause.getMessage();
core/src/main/java/org/testcontainers/utility/AuditLogger.java:import org.jetbrains.annotations.Nullable;
core/src/main/java/org/testcontainers/utility/AuditLogger.java:                             @Nullable String image,
core/src/main/java/org/testcontainers/utility/AuditLogger.java:                             @Nullable String containerId,
core/src/main/java/org/testcontainers/utility/AuditLogger.java:                             @Nullable String image,
core/src/main/java/org/testcontainers/utility/AuditLogger.java:                             @Nullable String containerId,
core/src/main/java/org/testcontainers/utility/AuditLogger.java:                             @Nullable Exception e) {
core/src/main/java/org/testcontainers/utility/AuditLogger.java:                                    @Nullable List<String> env) {
modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java:        Optional.ofNullable(resourceLocation)
modules/jdbc/src/main/java/org/testcontainers/jdbc/ConnectionUrl.java:        imageTag = Optional.ofNullable(urlMatcher.group(3));
modules/jdbc/src/main/java/org/testcontainers/jdbc/ConnectionUrl.java:            databasePort = Optional.ofNullable(dbInstanceMatcher.group(3)).map(value -> Integer.valueOf(value));
modules/jdbc/src/main/java/org/testcontainers/jdbc/ConnectionUrl.java:                                                Optional.ofNullable(urlMatcher.group(5)).orElse("")));
modules/jdbc/src/main/java/org/testcontainers/jdbc/ConnectionUrl.java:        initScriptPath = Optional.ofNullable(containerParameters.get("TC_INITSCRIPT"));
modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java:import org.jetbrains.annotations.Nullable;
modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java:    @Nullable
modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java:    @Nullable
```

but I guess it's the only place that uses javax.annotation.Nullable.  Looks like the rest use org.jetbrains.annotations.Nullable

**Comment by dbyron0** [comment]
For what it's worth, the test failures (timeouts) look unrelated to this change.

File: core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java [hunk 1]
```diff
@@ -233,9 +232,8 @@ public void put(InputStream body, com.github.dockerjava.core.MediaType mediaType
         execute(request).close();
     }
 
-    protected RequestBody toRequestBody(InputStream body, @Nullable String mediaType) {
+    protected RequestBody toRequestBody(InputStream body, String mediaType) {
```
Comment (rnorth) [inline core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java]: Maybe we should replace this with `org.jetbrains.annotations.Nullable`? At least this PR would get us to a consistent state, from which point we can consider whether it's worth keeping the jetbrains library in place!

Comment (dbyron0) [reply core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java]: Sure: https://github.com/testcontainers/testcontainers-java/pull/920/commits/1f5b77c65224fc448dc7773f2686bbee87d6e10a

**Comment by kiview** [comment]
I just merged with master, regarding the comment of @rnorth, I think we can merge this.

**Pull request merged**